### DISCUSSION TITLE
No longer possible to digitize a detailed surface of a wastewater structure

### DISCUSSION THREAD

**Discussion by rbovard** [teksi/wastewater#383] [Q&A] [answered]
As @ponceta has shown, it is no longer possible to digitize a detailed surface of a wastewater structure using the action in the current QGIS LTR (3.34).

What is the status of this bug, will it be resolved in a future 3.34 or does something need to be done in the Python function?

**Comment by ponceta** [comment, marked as answer]
Fixed on QGIS 3.34.8+ and later
